app/full_text_search.py

```diff
@@ -114,47 +98,74 @@ def create_romanized_query_txt(query_txt):
     return enclose_in_parenthesis(romanized_query)
 
 
-def get_person_ids_from_results(romanized_query, results_list):
+def is_query_match(query_txt, values, romanized_values):
+    """
+    To check if the query match the field value
+    Args:
+        query_txt: Search query
+        values: field value for unidecode method
+        romanized_values: romanized field value
+    Returns:
+        Boolean
+    """
+    if not query_txt:
+        return True
+
+    romanized_query_list = (script_variant.romanize_search_query(query_txt))
+
+    for name in romanized_query_list:
+        words = name.split(" ")
+        for word_index, word in enumerate(words):
+            if not re.search(word, values + " " + romanized_values, re.I):
+                break
+            if word_index == len(words) - 1:
+                return True
+    return False
+
+
+def get_person_ids_from_results(romanized_query_list, results_list):
     """
     Returns person record_id of persons
```

Comment:
The explanation is useful. If it's no longer accurate, please update it, but don't remove it